mail/models.py

```diff
@@ -0,0 +1,19 @@
+"""
+Models for mail
+"""
+from django.contrib.auth.models import User
+from django.db import models
+
+from financialaid.models import TimestampedModel, FinancialAid
+
+
+class FinancialAidEmailAudit(TimestampedModel):
+    """
+    Audit table for the Financial Aid
+    """
+    acting_user = models.ForeignKey(User, null=False)
+    financial_aid = models.ForeignKey(FinancialAid, null=True, on_delete=models.SET_NULL)
```

Comment:
I feel like there would be an integrity error due to the null constraint if the user deleted only `FinancialAid`. But I'm not sure so let's leave it then.